modules/retail/pages/retail-next.adoc

```diff
@@ -9,9 +9,9 @@ If you need further information or support, try one of these options.
 [[retail.sect.next.docs]]
 == More Documentation
 
-For {susemgr} documentation, visit https://www.suse.com/documentation/suse-manager-4/
+For {susemgr} documentation, visit https://documentation.suse.com/suma/4.0/
 
-For legacy {slepos} or {smr} documentation, see https://www.suse.com/documentation/slepos11/index.html or https://www.suse.com/documentation/suse-manager-for-retail.
+For legacy {slepos} or {smr} documentation, see https://documentation.suse.com/sle-pos/11-SP3/ or https://documentation.suse.com/suma-retail/3.2/.
 Note, however, that {smr} documentation supersedes this information.
```

Comment:
Drive-by finding: this is nicely unclear.

Reply:
Yeah, I was trying to avoid too much editing. I'll quickly split this into two, though.